service/src/main/java/org/cbioportal/genome_nexus/service/internal/MyVariantInfoServiceImpl.java

```diff
@@ -123,6 +123,17 @@ private String buildRequest(String variant)
         {
             sb.insert(0,"chr");
         }
+        // Check if the variant id contains "del", if so we will remove the deleted nucleotides
```

Comment:
Should we separate this logic into a utility method (under component module) and add some unit tests too?

Reply:
@onursumer done